assets/js/script.js

```diff
@@ -130,26 +128,39 @@ function checkResult() {
         }
     }
     isWinner(d1, d2);
-
     // Check for a draw
     if (emptyCells === 0) {
         document.getElementById("result").innerHTML = "It's a Draw!";
-        setTimeout(function () {
-            window.location.reload(); // Reload the window after a brief delay
-        }, 2000); // You can adjust the delay time (in milliseconds) as needed
```

Comment:
After draw window must be reload after a brief delay just like after winning condition

Reply:
Can you explicitly define the delay time?

Reply:
keep either 2000 or 3000
You have deleted the above code which causes the issue after draw